src/server/scripts/Pet/pet_warlock.cpp

```diff
@@ -7,37 +7,40 @@
  * Scriptnames of files in this file should be prefixed with "npc_pet_warlock_".
  */
 
-#include "ScriptedCreature.h"
-#include "ScriptMgr.h"
+/*#include "ScriptedCreature.h"
```

Comment:
Better to delete the entire file, you can see here how it should be done https://github.com/azerothcore/azerothcore-wotlk/pull/6087/commits/29bc6314b2b1607cb69637573cd71ac118498d38 

Good PR otherwise

Reply:
I prefer to leave the File there, so (IF) in the future we will need to add something, the file already exists and you just need to Edit.

Reply:
at least remove the script, also build fails because you commented out
```cpp
  void AddSC_warlock_pet_scripts()
{
}